app/models/user.rb

```diff
@@ -27,10 +27,11 @@ class User < ActiveRecord::Base
     where("name LIKE ? OR email LIKE ?", "%#{query}%", "%#{query}%")
   }
   scope :with_role, -> (role_id, project_id) {
-    joins('LEFT OUTER JOIN user_project_roles ON users.id = user_project_roles.user_id').
+    joins('LEFT JOIN user_project_roles ON users.id = user_project_roles.user_id').
```

Comment:
joins("LEFT OUTER JOIN user_project_roles ON users.id = user_project_roles.user_id AND user_project_roles.project_id = #{project_id.to_i}").
      where('users.role_id >= ? OR user_project_roles.role_id >= ?', role_id, role_id)